crabTask.py

```diff
@@ -396,9 +411,19 @@ def findOutputFile(self, taskOutput, jobId):
                          f' in {taskOutput}: ' + ' '.join(outputFiles))
     return outputFiles[0]
 
+  def getTarFileFromGfal(self, outputFile):
+    tempdir = os.path.join(os.environ['TMPDIR'], outputFile.split("/")[-1])
```

Comment:
Better to set a temporary directory in the yaml config because, in some sites, the size of TMPDIR area could be too small to copy large files. E.g.
```yaml
tmp_area: /path/to/tmp
```
or
```yaml
tmp_area: TMPDIR
```
then in the code:
```python
temp_root = os.environ['TMPDIR'] if self.tmp_area == 'TMPDIR' else self.tmp_area
```